codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -492,13 +476,12 @@ public void onGoAwayRead(ChannelHandlerContext ctx, int lastStreamId, long error
         public void onWindowUpdateRead(ChannelHandlerContext ctx, int streamId, int windowSizeIncrement)
                 throws Http2Exception {
             Http2Stream stream = connection.requireStream(streamId);
-            verifyGoAwayNotReceived();
-            if (stream.state() == CLOSED || shouldIgnoreFrame(stream, false)) {
-                // Ignore frames for any stream created after we sent a go-away.
+
+            if (stream.state() == CLOSED || streamCreatedAfterGoAwaySent(stream)) {
```

Comment:
I think we still need to count this toward flow control. http://http2.github.io/http2-spec/index.html#rfc.section.6.9:

> A receiver that receives a flow controlled frame MUST always account for its contribution against the connection flow control window, unless the receiver treats this as a connection error (Section 5.4.1). This is necessary even if the frame is in error. Since the sender counts the frame toward the flow control window, if the receiver does not, the flow control window at sender and receiver can become different.